src/widgets/clear-refinements/clear-refinements.js

```diff
@@ -0,0 +1,125 @@
+import React, { render, unmountComponentAtNode } from 'preact-compat';
```

Comment:
We should align the casing of the files with the rewrite. Components are pascal case, connectors camel case and the widget "kebab" case. IMO it's okay to have the components with a different case for the file but we should at least align the connectors & widgets. WDYT?

Reply:
Indeed we said that the file / folders should have the same case as the exported symbol. So it's gonna be camelCase then for widgets. Right?

Reply:
Yes. There is also the `/components/ClearRefinements/ClearRefinements.js` in that case `ClearRefinements` is a bit redundant. WDYT?

Reply:
I don't know. On one hand, it's nicer, but on the other it brings little value and the test are more easy to find with a nested structure.

Reply:
We can do that in another PR if we decide to move forward. Ok @samouss ?

Reply:
Sure!